### PR TITLE
Updated version to 2.4.0

### PR DESCRIPTION
**Pull request by HassanNiazi**
### Dependencies
  - https://github.com/Crana-Inc/hasura-service/pull/240
  - https://github.com/Crana-Inc/web-client/pull/568/
  - https://github.com/Crana-Inc/api-service/pull/53

### CODE REVIEW

**Comment by HassanNiazi** [comment]
Don't merge this; Infact fetch from upstream, and close this one once story is merged.

**Review by vladdy-dotcom — APPROVED** [review]

File: Dockerfile
Comment (vladdy-dotcom) [inline Dockerfile:1]: 2.5 is out

**Review by vladdy-dotcom — CHANGES_REQUESTED** [review]

**Review by vladdy-dotcom — APPROVED** [review]

**Pull request merged**
